data/json/recipes/recipe_deconstruction.json

```diff
@@ -1695,6 +1762,12 @@
     "time": "1 m",
     "components": [ [ [ "ceramic_shard", 1 ] ] ]
   },
+  {
+    "result": "ceramic_mug",
+    "type": "uncraft",
+    "time": "1 m",
+    "components": [ [ [ "ceramic_shard", 1 ] ] ]
```

Comment:
The ceramic mug should only have 12ml of components while 1 ceramic shard is 250ml. A single ceramic shard is also nearly 3 times as heavy as a ceramic mug!

Reply:
I just included the ceramic_mug because the bowl, cup and plate had the exactly same disassemble and are roughly the same weight. Then should we chance the disassemble of all of them?

Reply:
All of them seem broken - maybe just increase the volume of the ceramic mug slightly. I'll see if I can find the RL guidelines are for ceramic items

Reply:
I did a quick research and there is a huge variety for the "ceramics", from the basic ones that we use everyday such as porcelain to the most technological ones like the boron carbide, then the density can vary from 2.4g/ml to 6g/ml. The ceramic shard from CDDA uses a 3g/ml but the base_ceramic_dish don't use even 1.5g/ml, so I will change it but it will impact on every ceramic item from dining_kitchen.json so I will change all of them.